sys/net/crosslayer/ng_netif/ng_netif_addr_parse.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ */
+
+#include <string.h>
+
+#include "net/ng_netif.h"
+
+static uint16_t _parse_byte(char *str)
```

Comment:
IMHO it's not that straightforward... Yes, I figured it out after reading a few lines further, but handling an error like that seems a bit unusual to me, so grasping it may take a bit. In fact, the more I think about it… Is there a reason (other than being able to return UINT16_MAX) you're using uint16_t instead of uint_8? Because having something like
``_parse_byte(char *in, uni8_t *out)``  
which returns an error code and writes values to ``out`` on success seems more intuitive to me, would stop my bickering about documentation and save you the cast in line 55. ^^